### DISCUSSION TITLE
Current projects - where to get started?

### DISCUSSION THREAD

**Discussion by joshkadis** [WebView-CG/Compatibility-Data-Project#35] [General]
👋  @NiklasMerz  and others. I'm excited to join this group after several years of working with WebViews, specifically WKWebView in The Wall Street Journal's iOS and iPad app. Looking over the group's different repos and projects, it's tough to tell where it would be most helpful to contribute. What do you think?

**Comment by NiklasMerz** [comment]
Hi @joshkadis welcome to group. I'm excited to introduce every new member. If you haven't checked it out, yet a good place to start would be the summary of what we did this year: https://caniwebview.com/news/2024-12-06-WebView-Community-Group-Recap/

Our focus was mostly bringing data and features to caniwebview.com. If you have any ideas to improve it or you can imagine to write documentation check out the repo: https://github.com/WebView-CG/Compatibility-Data-Project. Having real-life experiences from apps like the Wall Stree Journal would be very much appreciated.

We are planning to host the next group meeting in January and I would love to welcome you there. Please go to the time poll to help us find a good meeting time: https://bitpoll.de/poll/webviewcg-25-1/

If you want to reach out you can mail the public mailing list or me personally if you like.

Hope to see you next year and kind regards
Niklas Merz

**Comment by NiklasMerz** [comment]
I started working on a few things with more focus just recently. Therefore I track my work in this project: https://github.com/orgs/WebView-CG/projects/1/views/1

If you want to add or help with something. Let me know.
